package.json

```diff
@@ -70,7 +70,7 @@
     "express": "^4.15.2",
     "gh-pages": "^1.0.0",
     "http-server": "^0.9.0",
-    "jest": "^19.0.2",
+    "jest": "^20.0.4",
```

Comment:
My bad, I was going crazy with some Jest syntax and lack of doc. Let's be more clean yeah next time ;)